syntax = "proto3";
import "PresenceState.proto";

message PresenceStatus {
  PresenceState presence = 1;
}